src/components/keymap/edit/keymap-edit.component.ts

```diff
@@ -10,17 +10,18 @@ import 'rxjs/add/operator/switchMap';
 import { Observable } from 'rxjs/Observable';
 import { ConnectableObservable } from 'rxjs/observable/ConnectableObservable';
 
-import { Keymap } from '../../config-serializer/config-items/Keymap';
-import { AppState } from '../../store';
-import { getKeymap } from '../../store/reducers/keymap';
+import { Keymap } from '../../../config-serializer/config-items/Keymap';
+import { AppState } from '../../../store';
+import { getKeymap } from '../../../store/reducers/keymap';
 
 @Component({
-    selector: 'keymap',
-    template: require('./keymap.component.html'),
-    styles: [require('./keymap.component.scss')]
+    selector: 'keymap-edit',
+    template: require('./keymap-edit.component.html'),
+    styles: [require('./keymap-edit.component.scss')]
 })
-export class KeymapComponent {
+export class KeymapEditComponent {
     private keymap$: Observable<Keymap>;
+    private keymapDeleted: boolean = false;
```

Comment:
What is the purpose of this member? It is just set in the onKeymapDelete, but otherwise it seems to me it is unnecessary.